doc-content/enterprise-only/admin-and-config/setup-auto-cleanup-proc.adoc

```diff
@@ -4,16 +4,20 @@
 You can set up an automatic cleanup job in {CENTRAL}.
 
 .Procedure
-. In {CENTRAL}, go to *Deploy* > *Jobs*.
+. In {CENTRAL}, go to *Manage* > *Jobs*.
 . Click *New Job*.
-. Enter a name, due date, time, and the following command into the *Type* text field.
+. Enter a *Business Key*, *Due On*, *Retries*, and the following command into the *Type* field.
```

Comment:
I suggest a minor rewrite to be a bit more concise.
. Enter values for the *Business Key*, *Due On*, and *Retries* fields.
. Enter the following command into the *Type* field.

Reply:
Done